platform_versioned_docs/version-23.4.0/data/data-studios.mdx

```diff
@@ -102,31 +108,53 @@ Once connected, the data studio session list will update to display the status o
 
 :::note
-Collaborators will need valid permissions to connect to the running data studio (at least the **Connect** role for the workspace).
+Collaborators need valid workspace permissions to connect to the running data studio.
 :::
 
-### Stop a data studio
+## Stop a data studio
 
-To stop a running session, select the three dots next to the status message and then select **Stop**. The status will change from **running** to **stopped**. When a data studio session is stopped, the compute resources it's using are deallocated and any unsaved state is lost. You can stop a session at any time.
+This functionality is available to all user roles with the **Maintain** role and above.
+
+To stop a running session, select the three dots next to the status message and then select **Stop**. The status will change from **running** to **stopped**. When a data studio session is stopped, the compute resources it's using are deallocated. You can stop a session at any time.
+
+Stopping a running session creates a new checkpoint.
 
 :::note
 Make sure you save before you stop a data studio.
 :::
 
-### Restart a stopped data studio
+## Restart a stopped data studio
+
+This functionality is available to all user roles with the **Maintain** role and above.
+
+When you restart a stopped session, a new session with the same configuration is created, using the most recent checkpoint.
 
-When you restart a stopped session, a new session with the same configuration is created. Note that none of the data from previous sessions is preserved unless saved to the mounted data.
+## Start a new data studio from a checkpoint
 
-### Delete a data studio
+This functionality is available to all user roles with the **Maintain** role and above.
+
+You can start a new data studio from an existing stopped data studio. This will inherit the history of the parent data studio checkpoint state. From the list of **stopped** data studios in your workspace, select the three dots next to the status message for the data studio and select **Start as new**. Alternatively, from the data studio detail page select the **Checkpoints** tab and choose a non-running checkpoint to start a new data studio from by clicking the three dots in the `Actions` column and selecting **Start as new**.
+
+## Delete a data studio
+
+This functionality is available to all user roles with the **Maintain** role and above.
 
 You can only delete a data studio when it's **stopped**. Select the three dots next to the status message and then select **Delete**. The data studio is deleted immediately and can't be recovered.
 
-### Data studio statuses
+## Data studio checkpoints
+
+When you start a newly-added data studio for the first time, it automatically creates a *checkpoint*. A checkpoint saves changes that you make to the data studio root filesystem and stores it in the compute environment's pipeline work directory.
```

Comment:
Clarification: A checkpoint is created every time a data studio session starts, no matter whether it is existing or newly-added, or for the first time :)